README.md

```diff
@@ -45,7 +45,7 @@ five.upHigh() // ⁵
 five.downLow() // ₅
 five.tooSlow() // 5, with a ~500 millisecond delay
 five.roman() // V
-five.morseCode() // di-di-di-di-dit
+five.morseCode() // di-di-di-di-dah
```

Comment:
The generally accepted way to pronounce Morse is with "dit"s and "dah"s, and a non-final "dit" is shortened to "di-".  http://www.acsu.buffalo.edu/~maxwell/Fundamental_Morse.html